packages/frontend/src/components/map/Map.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+import React, { useEffect, useState } from 'react';
+import { MapContainer, Marker, TileLayer, Popup } from 'react-leaflet';
+import 'leaflet/dist/leaflet.css';
+import 'tailwindcss/tailwind.css';
+import { LatLngTuple } from 'leaflet';
+
+function Map() {
+  // init variables for map
+  const attribute =
+    '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors';
+  const url = 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
+  const [userPosition, setUserPosition] = useState<LatLngTuple | null>(null);
+
+  // Get user position
+  const getPosition = () => {
```

Comment:
**nitpick (non-blocking):** We will fix issues like this to avoid lint warnings in the future